helm/bankapp/templates/configMap.yml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.configmap.name }} 
+  namespace: {{ default "bankapp-namespace" .Values.namespace }}
+data:
+  MYSQL_DATABASE: {{ .Values.configmap.data.MYSQL_DATABASE }} 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)